wordpress-mobile/gutenberg-mobile
# Recover border color on focused blocks

Small PR that fixes an issue where the border color for focused blocks were not being shown.

`gutenberg` PR: https://github.com/WordPress/gutenberg/pull/17269

![border](https://user-images.githubusercontent.com/9772967/64008078-adac1b00-cb15-11e9-8fdb-6cebfaee2575.png)

To test:
- Run the example app.
- Select a block.
- Check that the grey borders appear on the selected block.
